benchmarking/cli/launch_hpo.py

```diff
@@ -396,10 +396,10 @@
                 estimator_kwargs['max_run'] = int(1.01 * scheduler_timeout)
             log_level = logging.DEBUG if params['debug_log_level'] \
                 else logging.INFO
-            root_path = Path(__file__).parent.parent
+            root_path = repository_root_path()
             dependencies = [
                 str(root_path / module)
-                for module in ("benchmarks", "blackbox_repository")]
+                for module in ("benchmarking",)]
```

Comment:
Question to David: Since blackbox_repository is now under benchmarking, and so is utils, it is sufficient to have benchmarking here?

We could also use "benchmarking/blackbox_repository", "benchmarking/utils".